aws-android-sdk-core/src/main/java/com/amazonaws/http/JsonResponseHandler.java

```diff
@@ -129,7 +136,7 @@ public JsonResponseHandler(Unmarshaller<T, JsonUnmarshallerContext> responseUnma
 
             awsResponse.setResult(result);
 
-            final Map<String, String> metadata = new HashMap<String, String>();
+            final Map<String, String> metadata = new HashMap<>();
```

Comment:
This change has to be reverted as our maven compile and install step uses Java 1.6 source compatibility and this would fail even though Android Studio Gradle Build succeeds. 

I will look into documenting this and probably adding a build step to our build.gradle to prevent accepting this suggestion from Android Studio.

Reply:
Ah, thanks! Will fix.